### DISCUSSION TITLE
Execute Cron tasks on server start

### DISCUSSION THREAD

**Discussion by go4cas** [pocketbase/pocketbase#4892] [Q&A] [answered]
Using the javascript `cronAdd()`, my scheduled tasks execute as expected, but the initial run will only happen based on the cron time setting. Is there any way to have cron jobs execute on server start, and then follow the normal cron schedule?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure if that is what you are looking for but you can simply call your function manually, eg.:
```js
function test() {
    console.log(123);
}

test(); // note that the app may not be bootstrapped yet

cronAdd("test", "* * * * *", test)
```

**Comment by go4cas** [comment]
Thanks, @ganigeorgiev!

I have moved the handlers to a separate file, and then `require` these in the `cron.pb.js` and `main.pb.js` (initial run in `onAfterBootstrap`).
